### PR TITLE
Documenting MongoDB vector search integration

### PR DESCRIPTION
**Pull request by brimoor**

### CODE REVIEW

File: docs/source/integrations/mongodb.rst [hunk 1]
```diff
@@ -0,0 +1,751 @@
...
+    fob.compute_similarity(
+        dataset, 
+        patches_field="ground_truth",
+        model="clip-vit-base32-torch",
+        embeddings="embeddings",  # the attribute in which to store the embeddings
+        backend="mongodb",
+        brain_key="mongodb_patches",
+    )
+
+.. note::
```
Comment (allenleetc) [inline docs/source/integrations/mongodb.rst:427]: This Note dups the note immediately above the `Create a patch similarity index` example

**Review by allenleetc — APPROVED** [review]
LGTM, one minor comment

File: docs/source/integrations/mongodb.rst (hunk 1 above)
Comment (brimoor) [reply docs/source/integrations/mongodb.rst:427]: does it? this one is about extra kwargs, while the one above is basically saying "umm, yeah this section doesn't actually work yet!"

Comment (allenleetc) [reply docs/source/integrations/mongodb.rst:427]: All good, by the "one above" I meant the note above the "doesn't work yet". Nbd v minor

**Pull request merged**